src/pyodide-py/pyodide/console.py

```diff
@@ -150,4 +218,49 @@ def banner(self):
         """ A banner similar to the one printed by the real Python interpreter. """
         # copyied from https://github.com/python/cpython/blob/799f8489d418b7f9207d333eac38214931bd7dcc/Lib/code.py#L214
         cprt = 'Type "help", "copyright", "credits" or "license" for more information.'
-        return f"Python {platform.python_version()} {platform.python_build()} on WebAssembly VM\n{cprt}"
+        version = platform.python_version()
+        build = f"({', '.join(platform.python_build())})"
+        return f"Python {version} {build} on WebAssembly VM\n{cprt}"
+
+
+def repr_abreviate(
```

Comment:
I think `abreviate`might not be the best term. Maybe either `repr_shorten` (there is a https://docs.python.org/3.4/library/textwrap.html#textwrap.shorten which is very similar) or `repr_truncate` ?

Reply:
`repr_shorten` looks good to me.